features/main/hud/src/main/res/drawable/ic_baseline_music_note_24.xml

```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Looks like this is actually a duplicate of that image resource I commented on elsewhere - neither is in the correct location though, so please delete both and add a new one in the right place